src/scf-release/packages/suse-fuse/packaging

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+set -e -x
+
+DST=${BOSH_INSTALL_TARGET}
+SRC=/var/cache/zypp/packages
+
+# Clear the rpm cache. We want to see later only new files.
+zypper clean
+
+# Get fuse rpm and dependencies
+zypper --non-interactive install --download-only fuse
```

Comment:
I'm a bit of two minds here:

1. I think we should specify the precise package version here. A bosh release should always compile to the same bits and not incorporate whatever is latest at the time it is compiled. The Ubuntu fuse packages are embedded in the bosh release and don't change when Ubuntu updates the package.

2. Hard-coding the fuse version is a maintenance headache. We would have to update the bosh release whenever there is a CVE to this package, and we don't have a mechanism to notify us. However, the situation is only accidentally better by not hard-coding the version: we only rebuild when there is a new release of the full product.

In the end this is an issue produced by upstream. It is simply wrong to have stemcell-specific bits inside a bosh release. ☹️

Reply:
I can see both sides and am also two-minded. No decision yet. Lets see if in the end the fuse version I get differs for opensuse and sle setups.

Reply:
outdated with the move of fuse install to stemcells.